pyproject.toml

```diff
@@ -29,8 +29,10 @@ flask = "^1.1"
 pygal = "^2.4.0"
 Pygments = "^2.9.0"
 python-dateutil = "^2.8.2"
+cffi = "^1.14.6"
 
 [tool.poetry.dev-dependencies]
+"datanommer.models" = {path = "../datanommer/datanommer.models", develop = true}
```

Comment:
You can't do that because you don't know if the datanommer repo will be checked out in the same directory as the datagrepper repo. It's not the case in CI, for example.

Reply:
any ideas on alternatives here?

if i dont have this here, datagrepper uses the version of datanommer.models from pip (which we don't want)

Reply:
Using datanommer.models as a git dep here is not going to work i dont think either:

it is an open feature request with a WIP PR on poetry:
https://github.com/python-poetry/poetry/issues/755

Reply:
maybe a solution here is to start releasing some pre-release versions of datanommer to pypi so we can use something like this in datanommer for now:

```
[tool.poetry.dependencies]
"datanommer.models" = {version = "^0.9.2", allow-prerelease = true}
```

that way, we dont even have to checkout datanommer, can just use the prerelease version maybe?

Reply:
or retire the datagepper repo and make datagrepper a module similar to models, consumer, and commands, i,e. 
datanommer.datagrepper in the datanommer repo?

This is obviously the most drastic idea lol

Reply:
Yeah I think you could install manually `datanommer.models` using `poetry install` with the `poetry` exec from the virtualenv, then when you install datagrepper it won't try to install it because it's already there. That may work. Or maybe not if poetry really wants to install a specific version of datanommer.models, in which case you can do the manual install after datagrepper is installed.

Releasing a pre-version seems like a good idea too.

Reply:
okay, that approch works! thanks